testing/testdata/testmodels2.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+package testmodels;
```

Comment:
For consistency, let's make this `testdata`, too.

Reply:
Do we keep the custom data which allows for differentiation between proto2 and 3 or do you prefer to use predefined types as suggested in https://github.com/fullstorydev/grpcui/pull/95#discussion_r518776961.

I think there's some benefit to having tests for proto 2 and 3, if 2 is dropped at some point in the future failing tests can help signal it.

Reply:
>  if 2 is dropped at some point in the future

That can't realistically happen since `descriptor.proto`, which is at the very heart of all dynamic protobuf and RPC handling and even at the heart of protoc itself (as that is the format used by all protoc plugins), is proto2 (and must be, since it makes heavy use of extensions for custom options).

Reply:
Removed the custom messages completely